src/components/ecosystems/Collections/CollectionsContainer.jsx

```diff
@@ -185,6 +186,31 @@ export default compose(
           })
       }
     },
+    duplicateItem: ({ collectionsApiRoute, jekyll, branch, match, activeItem, updateCachedTree, history, collectionsRoute }) => () => {
+      const { collection } = match['params']
+    
+      if (collection && branch) {
+    
+        return jekyll.collection(collection, { ref: branch })
+          .then(collection => {
+            const duplicatedItem = activeItem
+            duplicatedItem['name'] = `${duplicatedItem['name'].split('.')[0]}-copy.md`
+    
+            const message = `[HT] Duplicated item: ${activeItem.path}`
+    
+            updateCachedTree()
+    
+            return collection.createItem(duplicatedItem, { ref: branch, message }).then(item => {
+              console.log('done creating item', item)
+              return Promise.resolve(item)
+                .then(
+                  history.push(`/${collectionsRoute}/${duplicatedItem['name']}?path=_${collection['name']}/${duplicatedItem['name']}&ref=${branch}`)  
```

Comment:
@heislercreative is the `?path=` parameter required? If so, it should be using the `item['path']` value, not collection name / item name because it is possible that collections live in a nested path depending on the jekyll config

Reply:
@5290charlie It appears the `?path=` parameter is not required. Corrected to simply `history.push(`/${collectionsRoute}/${item['name']}`)`